{"uuid": "693e936f-a783-44fb-870d-8efacf9abe71", "stdout": "fatal: [192.168.122.202]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.122.202 port 22: No route to host\\r\\n\", \"unreachable\": true}", "counter": 6, "pid": 20175, "event_data": {"play_pattern": "all", "play": "all", "task": "Gathering Facts", "task_args": "_ansible_version=2.5.2, _ansible_selinux_special_fs=['fuse', 'nfs', 'vboxsf', 'ramfs', '9p'], _ansible_no_log=False, gather_timeout=10, _ansible_module_name=setup, _ansible_verbosity=0, _ansible_syslog_facility=LOG_USER, _ansible_socket=None, gather_subset=all, _ansible_diff=False, _ansible_debug=False, _ansible_shell_executable=/bin/sh, _ansible_check_mode=False, _ansible_tmpdir=None", "remote_addr": "192.168.122.202", "res": {"msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.122.202 port 22: No route to host\r\n", "unreachable": true, "changed": false}, "pid": 20175, "play_uuid": "54e1ada8-887a-473b-b95f-000000000008", "task_uuid": "54e1ada8-887a-473b-b95f-000000000010", "playbook_uuid": "be2e5639-cc4f-45ef-99d0-41f0d1149b78", "playbook": "main.yml", "task_action": "setup", "host": "192.168.122.202", "task_path": "/home/dpivonka/Documents/ansible_runner_testing/ansible/project/main.yml:3"}, "end_line": 7, "created": "2018-06-19T17:29:38.962739", "start_line": 6, "event": "runner_on_unreachable"}